src/marketplace/DecentralandMarketplaceEthereum.sol

```diff
@@ -84,4 +144,38 @@ contract DecentralandMarketplaceEthereum is DecentralandMarketplaceEthereumAsset
 
         erc721.safeTransferFrom(_from, _asset.beneficiary, _asset.value);
     }
+
+    /// @dev Transfers MANA to the beneficiary depending to the provided value in USD defined in the asset.
+    function _transferUsdPeggedMana(Asset memory _asset, address _from) private {
+        // Obtains the price of MANA in ETH.
+        int256 manaEthRate = _getRateFromAggregator(manaEthAggregator, manaEthAggregatorTolerance);
+
+        // Obtains the price of ETH in USD.
+        int256 ethUsdRate = _getRateFromAggregator(ethUsdAggregator, ethUsdAggregatorTolerance);
+
+        // With the obtained rates, we can calculate the price of MANA in USD.
+        int256 manaUsdRate = (manaEthRate * ethUsdRate) / 1e18;
+
+        // Updates the asset with the new values.
+        _asset = _updateAssetWithConvertedMANAPrice(_asset, manaAddress, manaUsdRate);
```

Comment:
maybe `value` is a better name rather than `_asset`

Reply:
I'm updating the Asset object, which internally contains the value